### PR TITLE
Update CI to follow MSG and post compact columns

### LINKED ISSUE #543: The dataops-ci-bot should show the results of new schemas
When a pull request with a new schema is added, it is not possible to see what the generated schema looks like. Take for example the following snippet:

> 
> #### `bq_schema_88149d0-72f511d.diff`
> <details>
> <summary>Click to expand!</summary>
> 
> ```diff
> Only in integration/72f511d: pioneer-debug.debug.1.bq
> ```
> </details>

_Originally posted by @dataops-ci-bot in https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/542#issuecomment-624946365_

This contains a line with `Only in integration/72f511d: pioneer-debug.debug.1.bq`. It would be useful to have a link to the generated schema, or a full diff that includes the fully body (when it is under a certain text size).

### PR DESCRIPTION
**Pull request by acmiyaguchi**
This should fix #543 by using the `--new-file` option for diff. This should also post diffs of the compact column view.

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title)
- [ ] If adding a new field, the field should have a description (see #576 for an example)
- [ ] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
- [ ] If the PR comes from a fork, trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional) and review the report posted in the comments.

For glean changes:
- [ ] Update `templates/include/glean/CHANGELOG.md`

### CODE REVIEW

**Comment by acmiyaguchi** [comment]
The output of `mps bigquery diff` is available in the artifact tab of the circleci page now.

File: mozilla_pipeline_schemas/bigquery.py [hunk 1]
```diff
@@ -1,26 +1,34 @@
+import argparse
 import json
 import os
 import shutil
-import tempfile
-import argparse
```
Comment (fbertsch) [inline mozilla_pipeline_schemas/bigquery.py:5]: hm, wouldn't `black`  have found this? Can you run with black?

File: mozilla_pipeline_schemas/cli/bigquery.py [hunk 2]
```diff
@@ -75,4 +64,28 @@ def diff(base_ref, head_ref, input_directory, output_directory):
         schemas_path, head_ref, base_ref, integration_path
     )
 
-    write_schema_diff(head_rev_path, base_rev_path, integration_path)
+    # also compute the columns for each of these
+    def write_compact(path: Path):
+        for p in path.glob("*.bq"):
+            out = p.parent / p.name.replace(".bq", ".txt")
+            doc = json.loads(p.read_text())
+            out.write_text("\n".join(compute_compact_columns(doc)))
```
Comment (fbertsch) [inline mozilla_pipeline_schemas/cli/bigquery.py:72]:
```suggestion
            bq_schema = json.loads(p.read_text())
            out.write_text("\n".join(compute_compact_columns(bq_schema)))
```

**Review by fbertsch — APPROVED** [review]
r+. Can you add the issue to the commit when you rebase?

File: mozilla_pipeline_schemas/bigquery.py (hunk 1 above)
Comment (acmiyaguchi) [reply mozilla_pipeline_schemas/bigquery.py:5]: `black` doesn't change imports, that's the job of `isort`.

**Pull request merged**